Source/Scene/Cesium3DTileset.js

```diff
@@ -440,28 +393,29 @@ define([
             var childrenLength = children.length;
             var child;
             var k;
-            var additiveRefinment = (t.refine === Cesium3DTileRefine.ADD);
+            var additiveRefinement = (t.refine === Cesium3DTileRefine.ADD);
+
+            if (t.hasTilesetContent && t.isReady()) {
```

Comment:
Add a comment above `requestContent` below that says we don't check the child's visibility or SSE before requesting it because, the child is semantically the same tile (with the same SSE and bounding volume).